Exercise_3_Sister_Vocab/strings.py

```diff
@@ -1,3 +1,12 @@
+"""
```

Comment:
fun trick with python doc strings in comments (at least in VS Code) is to start a comments:

```
#
```

then make a doc string (really just a multi-line string

```
#"""
```

press enter a couple times

PROFIT.

python doesn't have multi-line comments but this takes the edge off :)

Reply:
Cool, neat trick! Thanks!